config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration_adv.h

```diff
@@ -2487,7 +2487,7 @@
    * M912 - Clear stepper driver overtemperature pre-warn condition flag.
    * M122 - Report driver parameters (Requires TMC_DEBUG)
    */
-  //#define MONITOR_DRIVER_STATUS
+  #define MONITOR_DRIVER_STATUS
```

Comment:
This board used SoftwareSerial which may cause performance issues during prints.

Reply:
As I could find in reviews, this option decreases performance on 8-bit controllers only. On the other hand this parameter can help to avoid a number of problems.
Did you see any evidence of performance issues on STM32?